src/components/Header/Header.component.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import Searchbar from '../Searchbar/Searchbar.component';
+import ProfileDropdown from '../ProfileDropdown/ProfileDropdown.component';
+import logo from '../../assets/youtube-logo.png';
+import Toggle from '../Toggle/Toggle.component';
+
+const Navbar = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 5rem;
+  background-color: #f3f3f3cc;
+  backdrop-filter: blur(4px);
+  box-shadow: 0px 2px 8px rgba(18, 45, 69, 0.2);
+  z-index: 1;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+`;
+
+const Logo = styled.img`
+  height: 5rem;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+const Container = styled.div`
```

Comment:
This is are well-defined styled components, you can improve this by moving them to their own file, this way when you need to know about the styles of the components you go to that file, if you need to know something about the logic of the component you rea this file and it is not crowded with the styles